sources: document setting up a bridge with AWS SQS source (#57)
triggermesh/docs: docs/sources/awssqs.md

> # AWS SQS event source for Knative Eventing

Reviewer: The title can also be changed to match [existing doc pages](https://docs.triggermesh.io/sources/awscodecommit/) (no mention of Knative needed here):
```suggestion
# Event source for AWS SQS
```

The event source captures messages sent to a [AWS SQS queue](sqs-docs) and sends them as CloudEvents to an event sink.

## Prerequisites

### SQS queue

If you don't already have an AWS SQS queue, create one by following the instructions in the [Getting started with Amazon SQS][sqs-getting-started] guide.

### Amazon Resource Name (ARN)

A fully qualified ARN is required to uniquely identify the AWS SQS queue.

![SQS queue](../images/awssqs-source/sqs-queue.png)

> As shown in the above screenshot, you can obtain the ARN of a SQS queue from the AWS console. Alternatively you can also use the [AWS CLI][aws-cli]. The following command retrieves the ARN of a SQS queue named `MyQueue` in the `us-west-2` region.

Reviewer: c.f. previous comment at line 32 (I just reused the phrasing from https://docs.triggermesh.io/sources/awsdynamodb/ for consistency)

````suggestion
As shown in the above screenshot, you can obtain the ARN of a SQS queue from the AWS console. It typically has the following format:

```
arn:aws:sqs:{awsRegion}:{awsAccountId}:{queueName}
```

Alternatively you can also use the [AWS CLI][aws-cli]. The following command retrieves the ARN of a SQS queue named `MyQueue` in the `us-west-2` region.
````

```console
$ aws --region us-west-2 sqs get-queue-attributes --queue-url $(aws --region us-west-2 sqs list-queues --queue-name MyQueue | jq -r .QueueUrls[0]) --attribute-names QueueArn
{
    "Attributes": {

>         "QueueArn": "arn:aws:sqs:us-west-2:043455440429:MyQueue"

Reviewer: I'd use `123456789012` instead of our own account ID in examples.

    }
}
```

If you don't have the AWS CLI installed on your workstation, use the following template to compose a fully qualified ARN of a SQS queue.

```
arn:aws:sqs:{awsRegion}:{awsAccountId}:{queueName}

> ```

Reviewer: > If you don't have the AWS CLI installed

Maybe not super relevant for SQS since you showed above that the ARN could be obtained from the console. See my comment at line 17.
